manifests/o/Oracle/JDK/24/24.0.0.0/Oracle.JDK.24.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+PackageIdentifier: Oracle.JDK.24
+PackageVersion: 24.0.0.0
+InstallerType: wix
+Scope: machine
+InstallerSwitches:
+  InstallLocation: INSTALLDIR="<INSTALLPATH>"
+UpgradeBehavior: install
+Commands:
+- java
+- javac
+- javaw
+- jshell
+FileExtensions:
+- class
+- jar
+- java
+ProductCode: '{4357E090-E02D-5CB2-9C32-639375983B2B}'

```

Comment:
```suggestion
ProductCode: '{4357E090-E02D-5CB2-9C32-639375983B2B}'
ReleaseDate: 2025-02-06
AppsAndFeaturesEntries:
- DisplayName: Java(TM) SE Development Kit 24 (64-bit)
  ProductCode: '{4357E090-E02D-5CB2-9C32-639375983B2B}'
  UpgradeCode: '{64A3A4F4-B792-11D6-A78A-00B0D0200240}'
InstallationMetadata:
  DefaultInstallLocation: '%ProgramFiles%\Java\jdk-24'
```

Reply:
I believe the release was yesterday (`2025-03-18`), not in February. I've used `2025-03-18` in my revision.